sdcm/nemesis.py

```diff
@@ -365,6 +364,19 @@ def disrupt_nodetool_cleanup(self):
         cmd = 'nodetool -h localhost cleanup keyspace1'
         self._run_nodetool(cmd, self.target_node)
 
+    def disrupt_modify_table_comment(self):
+        self._set_current_disruption('ModifyTableProperties %s' % self.target_node)
+        comment = ''.join(random.choice(string.ascii_letters) for i in xrange(24))
+        cmd = "ALTER TABLE keyspace1.standard1 with comment = '{}';".format(comment)
+        self.target_node.remoter.run('cqlsh -e "{}" {}'.format(cmd, self.target_node.private_ip_address), verbose=True)
+
+    def disrupt_modify_table_gc_grace_sec(self):
+        self._set_current_disruption('ModifyTableProperties %s' % self.target_node)
+        gc_grace_seconds = random.choice(xrange(216000, 864000))
+        cmd = "ALTER TABLE keyspace1.standard1 with comment = 'gc_grace_seconds changed' AND" \
+              " gc_grace_seconds = {};".format(gc_grace_seconds)
+        self.target_node.remoter.run('cqlsh -e "{}" {}'.format(cmd, self.target_node.private_ip_address), verbose=True)
+
 
```

Comment:
1. Do we have more option of alter table that aren't intrusive for the load?
2. If we ALTER for adding a column during c-s write load, will it fail the c-s load?